### PR TITLE
css-canvas-medical | Moses Kim

### PR DESCRIPTION
**Pull request by mosesr-kim**
Screen Shot 2021-05-08 at 6 11 57 PM
Screen Shot 2021-05-08 at 6 12 10 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Moe! Something went wrong with your screenshot, I can only see the one for the top half of the page

File: exercises/css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+                 already made Canvas the fabric of their care.
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-1-2">
+            <div class="row-center">
+              <div class="city-city">
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/index.html:99]: `city-city`? How about `city-list`?

File: exercises/css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+.img-text {
+  color: white;
+  font-family: 'Avenir', sans-serif;
+  line-height: 2;
+  font-size: 16px;
+  margin-left: 15%;
+}
+
+.img-link {
+  margin-top: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:113]: These are some high margin values! Part of your problem is the use of percentages, you should keep margin and padding to `px` or `rem` wherever possible. Try that out, that may make this a little less dangerous. If you find that your values are still really high, check in with me and we can discuss if there is a better solution.

File: exercises/css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  border-radius: 5px;
+}
+
+.container-city {
+  border-bottom: rgb(230, 230, 230) 1px solid;
+}
+
+.container-city-2 {
+  width: 100%;
+  margin-top: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:133]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  font-size: 18px;
+  line-height: 2;
+}
+
+.city-city {
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  margin-top: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:154]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+  line-height: 2.5;
+  text-align: center;
+  margin-bottom: 25%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:157]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+.start-head {
+  font-family: 'Tiempos Headline', serif;
+  color: #6e6e6e;
+  margin-top: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:163]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+  font-family: 'Avenir', sans-serif;
+  font-size: 24px;
+  background-color: #294363;
+  border-radius: 5px;
+  padding: 15px 50px 15px 50px;
+}
+
+.start-link {
+  text-align: end;
+  margin-top: 15%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:186]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+.container-start {
+  border-bottom: #6e6e6e 1px solid;
+}
+
+.start {
+  margin-bottom: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:194]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+
+.container-end {
+  background-color: #525351;
+}
+
+.end-text {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+  line-height: 1.6;
+  margin-top: 10%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:205]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 10]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+.end-link-head {
+  font-family: 'Avenir', sans-serif;
+  color: white;
+}
+
+#canvas-white {
+  width: 25%;
+  margin-top: 20%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:223]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 11]
```diff
@@ -0,0 +1,243 @@
...
+}
+
+.col-1-2-end {
+  width: 50%;
+  margin-top: 15%;
+  margin-bottom: 15%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:229]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

File: exercises/css-canvas-medical/styles.css [hunk 12]
```diff
@@ -0,0 +1,243 @@
...
+  font-family: 'Avenir', sans-serif;
+  color: #6e6e6e;
+  font-size: 24px;
+  font-weight: 600;
+}
+
+.col-1-2-footer {
+  width: 50%;
+  margin-top: 15%;
+  margin-bottom: 15%;
```
Comment (codingcodymiller) [inline exercises/css-canvas-medical/styles.css:242]: Same here, this is a big measurement. Please adjust this to `px` or `rem`, and if it is too large of a unit, think about alternatives.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start on this! It looks like a lot of notes, but it's primarily around using percentage for margin and padding, any maybe considering ways you can accomplish this with less need for margin and padding.

File: exercises/css-canvas-medical/styles.css (hunk 2 above)
Comment (mosesr-kim) [reply exercises/css-canvas-medical/styles.css:113]: done

File: exercises/css-canvas-medical/styles.css (hunk 4 above)
Comment (mosesr-kim) [reply exercises/css-canvas-medical/styles.css:154]: done

File: exercises/css-canvas-medical/styles.css (hunk 8 above)
Comment (mosesr-kim) [reply exercises/css-canvas-medical/styles.css:194]: done

File: exercises/css-canvas-medical/styles.css (hunk 5 above)
Comment (mosesr-kim) [reply exercises/css-canvas-medical/styles.css:157]: done

File: exercises/css-canvas-medical/styles.css (hunk 6 above)
Comment (mosesr-kim) [reply exercises/css-canvas-medical/styles.css:163]: done

**Review by codingcodymiller — APPROVED** [review]
Great work!!!

**Pull request merged**